### PR TITLE
Revamp directory structure

### PR DESCRIPTION
**Pull request by angelazhangmsft**
Renaming folders to follow the same naming conventions for all samples in this repo going forwards. Samples should be placed under a folder with the feature name (the "simple english name"), and further categorized by language/UI framework specific folders (for example Samples/TextRendering/cpp-win32)

Recommend navigating through the folders in this ranch to view the changes.

### CODE REVIEW

File: Samples/ResourceManagement/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ MRT Core is the modern resource loading library used in WinUI 3 and the Windows
 Before building the sample, make sure to set up your environment correctly by following the steps [here](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment).
 For the unpackaged sample apps, in addition to the steps for setting up the environment for the Windows App SDK, make sure to follow the instructions [here](https://docs.microsoft.com/windows/apps/windows-app-sdk/deploy-unpackaged-apps).
 
-1. Open the solution file (*.sln, e.g., `winui_desktop_packaged_app\winui_desktop_packaged_app.sln`) in Visual Studio 2019 Preview.
+1. Open the solution file (.sln) in the subfolder for your preferred sample in Visual Studio 2019 Preview.
```
Comment (rohanp-msft) [inline Samples/ResourceManagement/README.md:18]: >for [](http://example.com/codeflow?start=50&length=3)

of?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-- [Hello World sample](HelloWorld/reunioncppdesktopsampleapp): This sample demonstrates a basic integration with the Project Reunion NuGet package.
-- [DWriteCore gallery sample](DWriteCore/DWriteCoreGallery): This sample application demonstrates the DWriteCore API.
-- [MRT Core sample](MrtCore): This sample application demonstrates the MRT Core API.
+## Samples 
 
-For more information about the Windows App SDK, visit the [Windows App SDK Documentation](https://docs.microsoft.com/windows/apps/windows-app-sdk/). To learn more about the Windows App SDK design or to contribute to the project, make feature proposals, or start discussions, visit the [Windows App SDK GitHub page](https://github.com/microsoft/WindowsAppSDK).
+- [Hello World](Samples/HelloWorld): This sample demonstrates a basic integration with the Project Reunion NuGet package.
+- [Text Rendering](Samples/TextRendering): This sample is a gallery of the DWriteCore APIs demonstrating text rendering.
+- [Resource Management](Samples/ResourceManagement): This sample demonstrates app resource management using the MRT Core APIs.
```
Comment (rohanp-msft) [inline README.md:9]: >This sample [](http://example.com/codeflow?start=53&length=11)

These samples

File: README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 # Windows App SDK Samples
 
-This repository hosts samples related to the Windows App SDK. Currently it hosts the following three samples, and we will be adding a lot more as the project continues to grow.
+This repository hosts samples related to the [Windows App SDK](https://github.com/microsoft/WindowsAppSDK). Samples for various features shipping in the Windows App SDK will be added to this repository. For more information about the Windows App SDK, visit the [Windows App SDK Documentation](https://docs.microsoft.com/windows/apps/windows-app-sdk/). To learn more about the Windows App SDK design or to contribute to the project, make feature proposals, or start discussions, visit the [Windows App SDK GitHub page](https://github.com/microsoft/WindowsAppSDK).
```
Comment (rohanp-msft) [inline README.md:3]: >related to [](http://example.com/codeflow?start=30&length=10)

for

**Comment by rohanp-msft** [comment]
To be safe, it would be good to verify that there is no dependency anywhere on the name of the folder containing the solution and the project folders (since those names are changing in this PR).

File: DWriteCore/DWriteCoreGallery/DWriteCoreGallery.sln [hunk 4]
```diff
@@ -1,75 +1,75 @@
...
-		Release|Any CPU = Release|Any CPU
-		Release|ARM = Release|ARM
-		Release|ARM64 = Release|ARM64
-		Release|x64 = Release|x64
-		Release|x86 = Release|x86
-	EndGlobalSection
-	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{D6EC0907-F515-4F61-93EC-B89E19FA58B4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{D6EC0907-F515-4F61-93EC-B89E19FA58B4}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{D6EC0907-F515-4F61-93EC-B89E19FA58B4}.Debug|Any CPU.Deploy.0 = Debug|Any CPU
```
Comment (rohanp-msft) [inline DWriteCore/DWriteCoreGallery/DWriteCoreGallery.sln:26]: >Any CPU [](http://example.com/codeflow?start=72&length=7)

Intentional?

**Comment by angelazhangmsft** [comment]
> To be safe, it would be good to verify that there is no dependency anywhere on the name of the folder containing the solution and the project folders since those names are changing in this PR (I expect there to be no dependency).

Good point, off the top of my head there are links to individual samples in the various docs like [this](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/samples) which will need to be updated.

As for dependencies within the samples themselves, I have been verifying the samples all open and run as expected

File: Samples/ResourceManagement/cpp-winui/winui_desktop_packaged_app_cpp/readme.txt [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
-    under "Generated Files\sources".  
-3. Use the skeleton class definitions for reference to implement your
-    Windows Runtime classes.
-
-========================================================================
-Learn more about C++/WinRT here:
-http://aka.ms/cppwinrt/
-========================================================================
+========================================================================
+    C++/WinRT winui_desktop_packaged_app_cpp Project Overview
```
Comment (rohanp-msft) [inline Samples/ResourceManagement/cpp-winui/winui_desktop_packaged_app_cpp/readme.txt:2]: Should this file be an .md file too? (also, all the letters are lower case whereas in other cases, they're all upper case)

**Comment by rohanp-msft** [comment]
> This download contains Roboto Mono as both variable fonts and static fonts.

Should this file be an .md file too?

---
Refers to: Samples/TextRendering/cpp-win32/DWriteCoreGallery/RobotoMono/README.txt:4 in 785af83. [](commit_id = 785af83d5b7416588c7e26d0ca0be7ba41231678, deletion_comment = False)

File: Samples/ResourceManagement/cpp-winui/winui_desktop_packaged_app_cpp/readme.txt (hunk 5 above)
Comment (angelazhangmsft) [reply Samples/ResourceManagement/cpp-winui/winui_desktop_packaged_app_cpp/readme.txt:2]: This seems to be the automatic template file that comes with C++/WinRT projects - we can probably delete this file

**Review by rohanp-msft — APPROVED** [review]
Outside of my comments, looks good (y)

**Pull request merged**